spec/support/currently_undocumented_cops.txt

```diff
@@ -1,3 +1,5 @@
+RuboCop::Cop::Bundler::DuplicatedGem
```

Comment:
It looks like these new cops are documented in the rubocop codebase -- any idea why the scraper wasn't able to pull them in?